Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1115,6 +1115,20 @@ - (void)_enterFullScreen:(CGSize)mediaDimensions windowScene:(UIWindowScene *)wi
     }];
 }
 
+#if ENABLE(QUICKLOOK_FULLSCREEN)
+- (void)updateImageSource
+{
+    RetainPtr<WKWebView> webView = self._webView;
+    auto* manager = self._manager;
+    if (!manager)
+        return;
+
+    manager->prepareQuickLookImageURL([strongSelf = retainPtr(self), self, window = retainPtr([webView window]), logIdentifier = OBJC_LOGIDENTIFIER] (URL&& url) mutable {
```

Comment:
Can you remove the space between the capture list and the arg list? https://webkit.org/code-style-guidelines/#spacing-lambda-paren